threadpool/server/auth/src/main/java/cn/hippo4j/auth/toolkit/AESUtil.java

```diff
@@ -0,0 +1,92 @@
+package cn.hippo4j.auth.toolkit;
+
+import com.baomidou.mybatisplus.core.toolkit.IdWorker;
+import lombok.experimental.UtilityClass;
+
+import javax.crypto.*;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+/**
+ * Aes加解密算法工具类
```

Comment:
Please use English for comments in the code

Reply:
ok